src/randoop/HelperSequenceCreator.java

```diff
@@ -38,7 +38,7 @@
     } else {
       SimpleList<Sequence> candidates = components.getSequencesForType(cls.getComponentType(), false);
       if (candidates.isEmpty()) {
-        if (GenInputsAbstract.forbid_null) {
+        if (GenInputsAbstract.null_ratio == 0) {
           // No sequences that produce appropriate component values found, and null forbidden.
```

Comment:
The Randoop manual says of `--null-ratio`:  "If no non-null value is available, Randoop will try null regardless of the value of this parameter."  This code does not seem consistent with that statement.  (Do you think that Randoop's behavior should be different than what the manual says?)